transactor/src/main/scala/io/mediachain/datastore/DynamoDatastore.scala

```diff
@@ -15,10 +15,12 @@ class DynamoDatastore(config: DynamoDatastore.Config)
   
   val creds = config.awscreds
   val table = config.baseTable
-  val chunkSize = 1024 * 384 // 384 KB; DynamoDB has 400KB limit
+  val chunkSize = 1024 * 384 // 384 KB; DynamoDB has 400KB limit incl field
+                             // names
   val chunkTable = table + "Chunks"
-  
+
   val db = new AmazonDynamoDBClient(creds)
+  config.endpoint.foreach(ep => db.withEndpoint[AmazonDynamoDBClient](ep))
```

Comment:
[minor quip] just setEndpoint() -- why the fluid setter if you are discarding the value?